client/dashboard/task-list/tasks/payments/methods.js

```diff
@@ -71,6 +72,34 @@ export function getPaymentMethods( {
 				options.woocommerce_stripe_settings.enabled === 'yes',
 			optionName: 'woocommerce_stripe_settings',
 		},
+		{
+			key: 'wcpay',
+			title: __( 'WooCommerce Payments', 'woocommerce-admin' ),
+			content: (
+				<Fragment>
+					{ __(
+						'Accept credit card payments the easy way! No setup fees. No ' +
+							'monthly fees. Just 2.9% + $0.30 per transaction ' +
+							'on U.S.-issued cards',
+						'woocommerce-admin'
+					) }
+					{ __(
+						'By clicking "Setup," you agree to the Terms of Service',
```

Comment:
do we need to link to the Terms of Service here?

Reply:
Yup - forthcoming

Reply:
Done in https://github.com/woocommerce/woocommerce-admin/pull/3978/commits/a6eac31c48b08ec9effcf19ce7a117a78d8d6dcb